src/elements/content-sidebar/activity-feed/comment/Comment.js

```diff
@@ -161,7 +162,7 @@ class Comment extends React.Component<Props, State> {
         const createdAtTimestamp = new Date(created_at).getTime();
         const createdByUser = created_by || PLACEHOLDER_USER;
         const deleteConfirmMessage =
-            type === COMMENT_TYPE_DEFAULT ? messages.commentDeletePrompt : messages.taskDeletePrompt;
+            type === COMMENT_TYPE_DEFAULT ? messages.commentDeletePrompt : activityFeedMessages.taskDeletePrompt;
```

Comment:
Should `commentDeletePrompt` live in `/src/elements/content-sidebar/activity-feed/comment/messages.js`? It looks like the task-related messages live there, but not the comment-related ones, which seems odd.

Reply:
I split out the comment messages now too @jstoffan.